main.tex

```diff
@@ -490,50 +488,47 @@ \section{Results}\label{section:results}
-result of~\cite{Press2012} that against a memory-one opponent having a longer memory
-will not give a strategy any
-advantage. However, against two memory-one opponents Gambler's performance is better than
-the optimal memory-one strategy. This is evidence that in the case of two opponents having a
+than the best response memory-one strategy and in places outperforms it.
+However, against two memory-one opponents Gambler's performance is better than
+the optimal memory-one strategy. This is evidence that in the case of multiple
+opponents, having a
 shorter memory is limiting.
 
 \begin{figure}[!htbp]
     \centering
     \includegraphics[width=.5\textwidth]{img/gambler_performance_against_mem_one.pdf}
     \caption{The ratio between the utilities of Gambler and best response memory-one
     strategy for 152 different pair of opponents.}\label{fig:utilities_gambler_mem_one}
+    % TODO Nik: a legend for the lines or a description of the lines in the
+    % caption?
 \end{figure}
 
 \section{Discussion}
 This manuscript has considered \textit{best response} strategies in the IPD game, and
-more specifically, \textit{memory-one best responses}. It has proven that the utility
-of a memory-one strategy against a set of memory-one opponents can be written as a sum
-of ratios of quadratic forms and there is
-a compact way of identifying a memory-one best response to a group of opponents.
-The main results of this paper focused on a series of empirical results, where it
-was shown that the performance and the evolutionary stability of memory-one
-strategies rely on adaptability and not on extortion, and that
-memory-one strategies' performance is limited by their memory in cases where
-they interact with multiple opponents.
-
-Following the work described in~\cite{Nowak1989}, where it was shown that the
-utility between two memory-one strategies can be estimated by a Markov
-stationary state, we proved that the utilities can be written as a ration of two
-quadratic forms in $R^4$, Theorem~\ref{theorem_one}. This was extended to
-include multiple opponents, as the IPD is commonly studied in such situations.
-This formulation allowed us to introduce an approach for identifying memory-one
-best responses to any number of opponents;
-Theorem~\ref{memone_group_best_response} which does not only have game theoretic
-novelty, but also a mathematical novelty of solving quadratic ratio optimisation
-problems where the quadratics are non concave. Moreover, it allowed us to 
-obtaining a condition for which in an environment of memory-one opponents
+more specifically, \textit{memory-one best responses}.
+
+\begin{itemize}
+    \item It has proven that the utility
+          of a memory-one strategy against a set of memory-one opponents can be written as a sum
+          of ratios of quadratic forms (Theorem~\ref{theorem_one}).
+    \item  There is a compact way of identifying a memory-one best response to a
```

Comment:
Shouldn't this be?

```
more specifically, \textit{memory-one best responses}. It has proven that:

\item The utility...
\item There is a...
```

Reply:
I think I'd go with:


```

more specifically, \textit{memory-one best responses}. It was shown that:

\item The utility...
\item There is a...
```

Reply:
`There is a compact way of identifying a memory-one best response...` this was also proven right?

There is theorem proven this. But I don't mind shown either 👍